dnf/base.py

```diff
@@ -1064,6 +1066,16 @@ def download_packages(self, pkglist, progress=None, callback_total=None):
             percent = 100 - real / full * 100
             logger.info(msg, full / 1024 ** 2, real / 1024 ** 2, percent)
 
+        if self.conf.destdir:
+            dnf.util.ensure_dir(self.conf.destdir)
+            for pload in payloads:
+                payloadlocation = os.path.join(
+                    pload.pkg.repo.pkgdir,
+                    os.path.basename(pload.pkg.location)
+                )
+                shutil.copy(payloadlocation, self.conf.destdir)
+                os.chmod(os.path.join(payloadlocation, stat.S_IRWXO))
```

Comment:
@hoschiCZ During the testing I have found that the line does not change the result, but I can be wrong.